tests/qos/qos_sai_base.py

```diff
@@ -2137,6 +2154,14 @@ def select_port_ids_for_mellnaox_device(self, duthost, mgFacts, testPortIds):
         logger.info("Test ports ids is{}".format(test_port_ids))
         return test_port_ids
 
+    @pytest.fixture(scope="function", autouse=False)
+    def skip_src_dst_different_asic(self, dutConfig):
+        if dutConfig['dutAsic'] != dutConfig['dstDutAsic']:
```

Comment:
will hit KeyError: 'dstDutAsic'.
See WA in PR #10469

Reply:
The change in dutConfig adds key 'dstDutAsic' to it.